javacord-api/src/main/java/org/javacord/api/entity/message/Message.java

```diff
@@ -942,6 +942,16 @@ default Optional<CompletableFuture<Message>> requestReferencedMessage() {
      */
     Set<StickerItem> getStickerItems();
 
+    /**
+     * A generally increasing integer (there may be gaps or duplicates)
+     * Represents the approximate position of the message in a thread
+     * It can be used to estimate the relative position of the messsage
+     * in a thread in company with total_message_sent on parent thread.
```

Comment:
Even when placing dots after the sentences it reads very choppy so lets just change it to what's written in the docs
```suggestion
     * A generally increasing integer (there may be gaps or duplicates)
     * that represents the approximate position of the message in a thread,
     * it can be used to estimate the relative position of the message in a thread
     * in company with {@link ServerThreadChannel#getTotalNumberOfMessagesSent()} on parent thread.
```
This will probably not work right away because the mentioned method in the Javadoc does no yet exists. So we have to wait for #1089 to be merged first.

Reply:
Thanks for the putting suggested change :) !! Added the change but for now build will fail until #1089 is merged.

Reply:
will try to get as fast as possible